app/hooks/quest.tsx

```diff
@@ -155,11 +153,22 @@ export const useParsedJSXTextNickname = ({
     );
   }
 
+  const parseBoldText = (boldText: string) => (
+    <ExtendedText
+      key={boldText}
+      preset={preset}
+      // eslint-disable-next-line react-native/no-inline-styles
+      style={{ fontWeight: '700' }}>
```

Comment:
use `commonStyles` (create for bold text)
do not use inline styles even inside hooks